charts/gitea/values.yaml

```diff
@@ -20,6 +20,8 @@
 
 # Use edge termination for the gitea route
 tlsRoute: true
+# Default setting for insecure connections. Override if needed
+allowInsecure: None
```

Comment:
not sure if the name of this property is the best. its probably best to just name it `insecureEdgeTerminationPolicy` as `allowInsecure` isn't really matching to what the property is doing:
- https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html

thoughts @ckavili ?

Reply:
Thanks @garethahealy 
I have updated it